e2e_tests/scenarios/tests/common/api.rs

```diff
@@ -116,21 +116,32 @@ pub async fn inclusion_proof_raw(
     let result = String::from_utf8(bytes.into_iter().collect())
         .expect("Could not parse response bytes to utf-8");
 
-    Ok(RawResponse {
+    let raw_response = RawResponse {
         status_code: response.status(),
         body: result,
-    })
+    };
+
+    debug!(
```

Comment:
do we need it?

Reply:
It was helpful for me in general when looking what happened on errors. That's why it is on debug level - to not show it always. Do you think I should remove only that one or other debug logs there also?